sorc/machine-setup.sh

```diff
@@ -56,6 +56,25 @@ elif [[ -d /scratch1 ]] ; then
     export myFC=mpiifort
     export FCOMP=mpiifort
 
+##---------------------------------------------------------------------------
+elif [[ -d /data/prod ]] ; then
+    # We are on SSEC S4
+    if ( ! eval module help > /dev/null 2>&1 ) ; then
+	     echo load the module command 1>&2
+        source /usr/share/lmod/lmod/init/$__ms_shell
+    fi
+    target=s4
+
+    module purge
+    module load license_intel/S4
```

Comment:
Note this is going to change very soon as we move this all to the modulefiles where it belongs and convert them all to lua (see the work that was done on the ops branch for wcoss2). Not saying you have to change this right now, but that will be coming soon and we'll need you to do the conversion for S4.